code/game/objects/items.dm

```diff
@@ -1,4 +1,5 @@
 /obj/item
+	var/label_name = ""
```

Comment:
Still a huge no. 
Use replace text

Reply:
@TiviPlus I'd like your opinion on this one, please.

Reply:
Use a component like tg does